installer/bundle/bundle_skel.sh

```diff
@@ -41,6 +41,27 @@ INSTALL_TYPE=<INSTALL_TYPE>
 SCRIPT_LEN=<SCRIPT_LEN>
 SCRIPT_LEN_PLUS_ONE=<SCRIPT_LEN+1>
 
+# Error codes and categories:
+
+# User configuration/parameters:
```

Comment:
Looks good. Do you want to add prefix OMS to these constants to make them more descirptive? There are some SCX specific , there could be some OMI only and some of these seem Common errors.